docker/sudo-debian/Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM debian:wheezy
-MAINTAINER samvanoort@gmail.com
+FROM debian:buster
+MAINTAINER mark.earl.waite@gmail.com
 
-RUN apt-get update 
-RUN apt-get install -y sudo bc
-RUN useradd mysudoer -u @@MYUSERID@@
+ARG USER_ID
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y sudo bc curl default-jdk
```

Comment:
NIT: Would be nice to merge into the single command and cleanup temp files. But I do not care too much about non-production images

Reply:
###

Reply:
Yep this command will lead to more docker layer and bigger image as you keep apt metadata in your image, it's better to update, upgrade, install, clean in one command like [here](https://github.com/jenkins-infra/docker-packaging/blob/master/Dockerfile#L30L13)